code/modules/client/client procs.dm

```diff
@@ -362,14 +362,17 @@
 
 /client/proc/log_client_to_db()
 
-	if( IsGuestKey(src.key) )
+	if( IsGuestKey(key) )
 		return
 
+	// Isn't the ckey sanitized by default?
+	var/sql_ckey = sanitizeSQL(ckey)
```

Comment:
Yes, the ckey is effectively pre-sanitized by BYOND. Which is why I recently *removed* this. Why are you adding it back in?